plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSystemTables.java

```diff
@@ -189,5 +192,68 @@ public void testFilesTable()
                         "('split_offsets', 'array(bigint)', '', '')," +
                         "('equality_ids', 'array(integer)', '', '')");
         assertQuerySucceeds("SELECT * FROM test_schema.\"test_table$files\"");
+
+        MaterializedRow filesResult = computeActual("SELECT file_format, record_count, " +
+                "lower_bounds, upper_bounds, file_path FROM test_schema.\"test_table$files\" order by file_path").getMaterializedRows().get(0);
+
+        // file format
+        assertEquals(filesResult.getField(0), "ORC");
+        // record count
+        assertEquals(filesResult.getField(1), Long.valueOf(1));
+        // lower bound
+        assertEquals(filesResult.getField(2),
+                Map.of(Integer.valueOf(1), "0", Integer.valueOf(2), "2019-09-08"));
+        // upper_bound
+        assertEquals(filesResult.getField(3),
+                Map.of(Integer.valueOf(1), "0", Integer.valueOf(2), "2019-09-08"));
+    }
+
+    @Test
+    public void testAllFilesTable()
+    {
+        assertQuery("SHOW COLUMNS FROM test_schema.\"test_table$all_files\"",
+                "VALUES ('content', 'integer', '', '')," +
+                        "('file_path', 'varchar', '', '')," +
+                        "('file_format', 'varchar', '', '')," +
+                        "('record_count', 'bigint', '', '')," +
+                        "('file_size_in_bytes', 'bigint', '', '')," +
+                        "('column_sizes', 'map(integer, bigint)', '', '')," +
+                        "('value_counts', 'map(integer, bigint)', '', '')," +
+                        "('null_value_counts', 'map(integer, bigint)', '', '')," +
+                        "('nan_value_counts', 'map(integer, bigint)', '', '')," +
+                        "('lower_bounds', 'map(integer, varchar)', '', '')," +
+                        "('upper_bounds', 'map(integer, varchar)', '', '')," +
+                        "('key_metadata', 'varbinary', '', '')," +
+                        "('split_offsets', 'array(bigint)', '', '')," +
+                        "('equality_ids', 'array(integer)', '', '')");
+        assertQuerySucceeds("SELECT * FROM test_schema.\"test_table$all_files\"");
+
+        MaterializedRow allFilesResult = computeActual("SELECT file_format, record_count, " +
+                "lower_bounds, upper_bounds, file_path FROM test_schema.\"test_table$all_files\" order by file_path limit 1").getMaterializedRows().get(0);
+
+        // file format
+        assertEquals(allFilesResult.getField(0), "ORC");
```

Comment:
Same comment around comparing MaterializedResult

Reply:
thanks a lot @phd3 for the review!

are you referring to this [comment](https://github.com/trinodb/trino/pull/11206/files/0c7298acd28ce9ecdd19c54b09cdcc04162f1d90#r846390177) ?